portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<String>();
+    comments.add("Wow, this is a great website!");
+    comments.add("I enjoyed the pictures that you shared.");
+    comments.add("The tech that you use is awesome!");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+
+    Gson gson = new Gson();
+    String commentsjson = gson.toJson(comments);
```

Comment:
Can inline gson here

Reply:
I am not sure of what you mean by "inline gson"

Reply:
Sorry. Instead of declaring a variable gson, you could have

```
String commentsJson = (new Gson()).toJson(comments);
```

Also, just caught this, but also please capitalize the "j" in "commentsjson" so that it is in camel case.

Reply:
Oh ok, gotcha. Thanks for that clarification.